app/services/builders/chartkick/department_distribution_data.rb

```diff
@@ -0,0 +1,20 @@
+module Builders
+  module Chartkick
+    class DepartmentDistributionData < Builders::Chartkick::Base
+      def call
+        department_name = ::Department.find(@entity_id).name
+
+        review_turnarounds = retrieve_review_turnarounds
+        [{ name: department_name, data: build_distribution_data(review_turnarounds) }]
+      end
+
+      private
+
+      def retrieve_review_turnarounds
+        ::ReviewTurnaround.joins(review_request: { project: { language: :department } })
+                          .where(departments: { id: @entity_id })
+                          .where(created_at: @query[:value_timestamp])
```

Comment:
what is `@query[:value_timestamp]` ??

Reply:
I had to adapt the code to the old one. @query is a method in Base
```
      #app/services/metric/group/base.rb
      def query
        {
          name: @metric_name,
          value_timestamp: value_timestamp,
          interval: interval
        }
      end

      #app/services/metric/group/weekly.rb
      def value_timestamp
        (current_time - @prev.weeks).beginning_of_week..current_time.end_of_week
      end
```